tests/conf/i18n/translatehelpervisitor.js

```diff
@@ -12,11 +12,15 @@ async function createTranslator () {
   } } });
 }
 
-describe('TranslateHelperVisitor usage', () => {
-  it('static translation', async () => {
+describe('TranslateHelperVisitor translates visited nodes', () => {
+  let translator;
+  beforeEach(async () => {
```

Comment:
I guess I was thinking we don't need to make a new translator before each test, the translator should be able to handle translating multiple templates without reinstantiating itself. We should be able to have one translator used between all the tests

Reply:
Ok, just switched it to beforeAll instead